user-guide/Standard_Apps/EPM/EPM_Availability/Availability_deployment.md

```diff
@@ -0,0 +1,67 @@
+---
+uid: Availability_deployment
+keywords: Availability deployment
+---
+
+# Deploying the DataMiner EPM Availability solution
+
+To deploy the Availability branch of the EPM Solution:
+
+1. Deploy the EPM package:
+
+   - If the DMA is connected to dataminer.services, you can [deploy the package](xref:Deploying_a_catalog_item) directly from the Catalog.
```

Comment:
Could you add a link to the Catalog page here?

Reply:
added